src/reference/config/bindings/binding-http-kafka.md

```diff
@@ -203,18 +205,45 @@ routes:
       capability: fetch
       topic: items-snapshots
       filters:
-        - key: "${params.id}"
+        - key: ${params.id}
+  - when:
+      - method: POST
+        path: /items
+    exit: kafka_cache_client
+    with:
+      capability: produce
+      topic: items-crud
+      key: ${idempotencyKey}
+```
+
+Correlated Request-Response topics:
+
+```yaml
+routes:
   - when:
-      - path: /items/{id}
       - method: GET
-        path: /items/{id};{correlationId}
+        path: /items/{id}_{correlationId}
+    exit: kafka_cache_client
+    with:
+      capability: Fetch
+      topic: items-responses
+      filters:
+        - key: ${params.id}
+          headers:
+            - zilla:correlation-id: ${correlationId}
+  - when:
```

Comment:
the origional route below had the `GET` request on a route with the `produce` capability. Other places there has to be a `fetch` capability defined. Is the `GET` path only there to match the async location path?

```yaml
    routes:
      - when:
          - method: GET
            path: /items/{id};{correlationId}
          - path: /items/{id}
        exit: north_kafka_cache_client
        with:
          capability: produce
          topic: items-requests
          key: ${params.id}
          reply-to: items-responses
          async:
            location: /items/${params.id};${correlationId}
```